common/wforce-geoip2.cc

```diff
@@ -135,6 +135,93 @@ bool WFGeoIP2DB::mmdbLookup(const std::string& ip, MMDB_lookup_result_s& res)
   return false;
 }
 
+char* convertToCharStar(const std::pair<unsigned int, std::string>& pair)
+{
+  char *cs = new char[pair.second.size()+1];
+  std::strcpy(cs, pair.second.c_str());
+  return cs;
```

Comment:
`return std::strdup(pair.second.c_str())`?

Reply:
Yeah that would also work. Also I guess I'd need to use free() rather than delete[] (and keep the check for nullptr)